tests/ndarrays/test_ndarrays.c

```diff
@@ -178,6 +178,35 @@ int32_t test_indexing_int64(void)
     return (0);
 }
 
+int32_t test_indexing_int64_order_f(void)
+{
+    int64_t m_1[] = {1, 6, 6, 2, 4, 0, 1, 5,
+                    7, 1, 3, 5, 3, 6, 4, 9,
+                    7, 6, 7, 1, 7, 7, 4, 4,
+                    7, 1, 9, 0, 1, 2, 4, 5,
+                    6, 1, 7, 3, 1, 5, 9, 6};
+    int64_t m_1_shape[] = {5, 8};
+    t_ndarray x;
+    int32_t index;
+    int32_t c_index;
+    int64_t value;
+    int64_t c_value;
+
+    x = array_create(2, m_1_shape, nd_int64, false, order_f);
+    memcpy(x.raw_data, m_1, x.buffer_size);
+    // testing the index [3, 2]
+    index = 3 * x.strides[0] + 2 * x.strides[1];
+    c_index = 13;
+    my_assert(index , c_index, "testing the strides");
```

Comment:
This tests the strides not the index

Reply:
Yes, I tried to copy how the other tests were done, the strides are tested anyway (the index function is tested on the following line), adding strides tests here seems productive imo, since strides and indexing are related

Reply:
My point is that the comment says "testing the index"

Reply:
"testing the strides"? or do you mean that we should change the comment to "testing the index"?
```c  
    index = 3 * x.strides[0] + 2 * x.strides[1];
    c_index = 13;
    my_assert(index , c_index, "testing the strides"); // making sure that the strides are correct
    my_assert(get_index(x, 3, 2) , c_index, "testing the indexing function"); // checking that that 'get_index' is correct
```
The `get_index` function does use strides to calculate index anyway, so maybe it is counterproductive to check for them there, maybe we should just remove that line from all the tests
@bauom

Reply:
Line 197. The start of this code snippet:
```
// testing the index [3, 2]
```

Reply:
```suggestion
    // testing the strides for index [3, 2]
    index = 3 * x.strides[0] + 2 * x.strides[1];
    c_index = 13;
    my_assert(index , c_index, "testing the strides");
```

Reply:
oh right! mb